### PR TITLE
Ongoing feature README review

### PR DESCRIPTION
**Pull request by mmcshane**
## What was changed

Several READMEs added

## Why?

Documenting Temporal features

## Checklist

1. Closes 

2. How was this tested:

3. Any docs updates needed?

### CODE REVIEW

File: features/signal/external/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Signals can be to a workflow execution from non-workflow code
```
Comment (Sushisource) [inline features/signal/external/README.md:1]:
```suggestion
# Signals can be sent to a workflow execution from non-workflow code
```

File: features/signal/prevent_close/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Pending signals prevents workflows from closing
+
+In Temporal, any pending signal for a given workflow execution will prevent that
```
Comment (Sushisource) [inline features/signal/prevent_close/README.md:3]: Don't think we need "In Temporal" in front of all these

File: features/signal/prevent_close/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+This feature tests that a signal queued for processing at the time that a
+workflow would otherwise exit in fact keeps the workflow running until the
+signal has been delivered and processed.
+
+# Detailed spec
+
+In most cases, having run the workflow function to completion, the SDK will send
+a `RespondWorkflowTaskCompletedRequest` to the server containing a command of
+type `COMMAND_TYPE_COMPLETE_WORKFLOW_EXECUTION` to indicate that the workflow
```
Comment (Sushisource) [inline features/signal/prevent_close/README.md:14]: Or any other command which would complete workflow execution - failure and continue as new both apply too

File: features/signal/workflow_to_workflow/README.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Workflow-to-workflow signals
+
+In Temporal, signals can be sent from one workflow to another workflow in the
+same namespace via the language's Temporal SDK API.
+
+# Detailed spec
+
+Signals sent from within one workflow via the are transmitted to the target
+workflow execution and delivered for processing.
```
Comment (Sushisource) [inline features/signal/workflow_to_workflow/README.md:9]:
```suggestion
Signals sent from within one workflow via the workflow api are transmitted to the target
workflow execution and delivered for processing.
```

File: features/signal/prevent_close/README.md (hunk 2 above)
Comment (lorensr) [reply features/signal/prevent_close/README.md:3]: +1

File: features/child_workflow/handle/README.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+# Child workflow handle
+
+Starting a Child returns a handle.
+
+This feature verifies that handles only have these properties:
```
Comment (Sushisource) [inline features/child_workflow/handle/README.md:5]: Maybe should be have at least?

**Review by Sushisource — APPROVED** [review]
Gonna merge this so we have it in

**Pull request merged**